.github/workflows/js.yml

```diff
@@ -60,12 +60,12 @@ jobs:
         run: |
           # On Windows, `yarn install` changes hash of @databricks/design-system in yarn.lock.
           # Use `--no-immutable` to allow the change.
-          yarn install --no-immutable
+          yarn install --no-immutable --no-check-resolutions
           git diff
       - name: Install dependencies (non-windows)
         if: runner.os != 'Windows'
         run: |
-          yarn install --immutable
+          yarn install --no-immutable --no-check-resolutions
```

Comment:
cc @hubertzub-db with the upgrade to yarn 4.2.2, i kept running into this issue on ci:
https://yarnpkg.com/advanced/error-codes#yn0078---resolution_mismatch

it also looks like this command modifies the yarn.lock file in CI, but it doesn't seem to happen locally. just wondering if you have any context around why this might be happening, and whether it's safe to set these flags in CI.